lib/api/mapBenefits.js

```diff
@@ -81,16 +81,11 @@ export function MapEICard(eiBenefitData) {
           eiBenefitData.publishDate,
           eiBenefitData.messageId
         ),
+        FormatSummary(
+          SummaryTypes.EstimatedDecisionDate,
+          estimatedDateOfDecision.getTime()
+        ),
       ]
-      //remove estimated decision date if current day is past that point
-      if (Date.now() < estimatedDateOfDecision.getTime()) {
```

Comment:
I don't believe the fix was to remove this logic but to update the dates so they show... should confirm with Breanna

Reply:
Discussed at stand-up; check notes in ticket. Ask Breanna if clarification still needed.

Reply:
I don't think bumping every date is a good idea, a lot of it is in place to demo a scenario, you should confirm with Breanna if she's okay with that. My understanding is just the dates that effect the display of "Estimated Decision" for the "Application Revived" should change.

Reply:
Breanna said she's okay with bumping every date up